tx.go

```diff
@@ -14,6 +14,7 @@ const (
 type Direction string
 type Status string
 type TokenType string
+type TransactionType string
```

Comment:
would be good to add transactions types into constant?

Reply:
yes!! done!!